semantic_conventions/logs/general.yaml

```
@@ -0,0 +1,19 @@
+groups:
+  - id: log.record
+    prefix: log.record
+    brief: >
+      The attributes described in this section are rather generic. They may be used in any Log Record they apply to.
+    attributes:
+      - id: uid
+        type: string
+        requirement_level: optional
```

```suggestion
        requirement_level: opt_in
```

Since https://github.com/open-telemetry/opentelemetry-specification/pull/3228 was merged, we need to adapt now.